apps/aecore/src/aec_trees.erl

```diff
@@ -232,7 +233,17 @@ gc_cache(Trees, TreesToGC) ->
 perform_pre_transformations(Trees, Height) ->
     Trees0 = aect_call_state_tree:prune(Height, Trees),
     Trees1 = aeo_state_tree:prune(Height, Trees0),
-    set_ns(Trees1, aens_state_tree:prune(Height, ns(Trees1))).
+    Trees2 = set_ns(Trees1, aens_state_tree:prune(Height, ns(Trees1))),
+    case Height =:= 0 of
```

Comment:
Compare with `aec_block_genesis:height()` instead?

Reply:
I would consider treating 0 as a fork_height and just have one case statement?